internal/cmd/base/describe.go

```diff
@@ -27,7 +24,7 @@ type DescribeCmd struct {
 	JSONKeyGetByName string // e.g. "servers"
 	NameSuggestions  func(client hcapi2.Client) func() []string
 	AdditionalFlags  func(*cobra.Command)
-	Fetch            func(ctx context.Context, client hcapi2.Client, cmd *cobra.Command, idOrName string) (interface{}, *hcloud.Response, error)
+	Fetch            func(ctx context.Context, client hcapi2.Client, cmd *cobra.Command, idOrName string) (interface{}, interface{}, error)
```

Comment:
Could we add a comment to explain what the 2 interfaces are?

Reply:
And maybe we can create a `DescribeCmdFetchFunc` type to document the function and reduce the duplication for all the describe commands ?

```go
// DescribeCmdFetchFunc TODO
type DescribeCmdFetchFunc func(ctx context.Context, client hcapi2.Client, cmd *cobra.Command, idOrName string) (interface{}, interface{}, error)
```

Reply:
Added in 827819c9ce89f2db6dae0693f5c0d2adb83eceaa